File: src/main/java/ru/dkolmogortsev/messages/TaskStopped.java
package ru.dkolmogortsev.messages;

import griffon.core.event.Event;

/**
 * Created by dkolmogortsev on 2/25/17.
 */
public class TaskStopped extends Event
{
    public TaskStopped(long timeEntryId)
    {
        super(timeEntryId);
    }

    public long getId()
    {
        return (long)super.getSource();
    }
}

File: src/main/kotlin/ru/dkolmogortsev/messages/TaskStopped.kt
package ru.dkolmogortsev.messages

import griffon.core.event.Event

/**
 * Created by dkolmogortsev on 2/25/17.
 */
class TaskStopped(timeEntryId: Long) : Event(timeEntryId) {

    val id: Long
        get() = super.getSource() as Long
}
